assets/sass/_base.scss

```diff
@@ -1,20 +1,3 @@
-html {
-  height: 100%;
-}
-
-body {
-  font-family: $fonts;
-  font-size: $font-size;
-  font-weight: $font-weight-body;
-  line-height: 1.4;
-  background-color: $secondary-bg-color;
-  min-width: 100%;
-  min-height: 100%;
-  color: $medium-text-color;
-  position: relative;
-  overflow: auto;
-}
-
 a {
```

Comment:
yeah, I guess it doesn't make sense to create _container.scss at all. Everything can just be namespaced in _base.scss